no method named `extract` found for type `&&pyo3::PyInstance` in the current scope

```
error[E0599]: no method named `extract` found for type `&&pyo3::PyInstance` in the current scope
  --> src/lib.rs:11:1
   |
11 | #[py::modinit(_is_minified_js)]
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = help: items from traits can only be used if the trait is in scope
   = note: the following trait is implemented but not in scope, perhaps add a `use` for it:
           candidate #1: `use pyo3::ObjectProtocol;`

error: aborting due to previous error(s)
```

https://travis-ci.org/messense/is-minified-js/jobs/246491562

--- a/pyo3cls/src/module.rs
+++ b/pyo3cls/src/module.rs
@@ -266,6 +266,7 @@ fn wrap_fn(item: &mut syn::Item) -> Option<Box<syn::Block>> {
                     {
                         use std;
                         use pyo3 as _pyo3;
+                        use pyo3::ObjectProtocol;
 
                         #wrapper
 

